src/OpenLoco/src/GameCommands/ClearLand.cpp

```diff
@@ -11,17 +14,59 @@ namespace OpenLoco::GameCommands
     static loco_global<uint32_t, 0x00F0013C> _F0013C;
     static loco_global<uint32_t, 0x00F00140> _F00140;
     static loco_global<uint16_t, 0x00F00144> _F00144;
+    static loco_global<uint8_t*, 0x00F0016C> _F0016C; // stack offset to -eventually- flags
     static loco_global<uint16_t, 0x00F00170> _F00170;
+    static loco_global<uint32_t, 0x00F0013C> _F003CE; // tile x?
+    static loco_global<uint32_t, 0x00F00140> _F003D0; // tile y?
+    static loco_global<uint32_t, 0x00F25308> _F25308; // cost?
 
-    // 0x00469D76
-    static uint32_t sub_469D76(World::Pos2 pos, const uint8_t flags)
+    // 0x004690FC
+    static void sub_4690FC(World::Pos2 pos)
     {
         registers regs;
         regs.ax = pos.x;
         regs.cx = pos.y;
-        regs.bl = flags;
-        call(0x00469D76, regs);
-        return regs.ebx;
+        call(0x004690FC, regs);
+    }
+
+    // 0x00469D76
+    static uint32_t sub_469D76(World::Pos2 pos, const uint8_t flags)
+    {
+        // This shoudn't happen due to using TilePosRangeView
+        if (!World::validCoords(pos))
+        {
+            GameCommands::setErrorText(StringIds::off_edge_of_map);
+            return GameCommands::FAILURE;
+        }
+
+        _F25308 = 0;
```

Comment:
should call this _sub469E07Cost or something make it clear what it is there for.